include/functions.h

```diff
@@ -413,9 +413,9 @@ s32 func_8002DF54(GlobalContext* globalCtx, Actor* actor, u8 arg2);
 void func_8002DF90(DynaPolyActor* dynaActor);
 void func_8002DFA4(DynaPolyActor* dynaActor, f32 arg1, s16 arg2);
 s32 func_8002DFC8(Actor* actor, s16 arg1, GlobalContext* globalCtx);
-s32 func_8002E084(Actor* actor, s16 arg1);
-s32 func_8002E12C(Actor* actor, f32 arg1, s16 arg2);
-s32 func_8002E1A8(Actor* actorA, Actor* actorB, f32 arg2, s16 arg3);
+s32 Actor_YawInRangeWithPlayer(Actor* actor, s16 arg1);
+s32 Actor_YawAndDistInRangeWithPlayer(Actor* actor, f32 arg1, s16 arg2);
+s32 Actor_YawAndDistInRangeWithActor(Actor* actorA, Actor* actorB, f32 arg2, s16 arg3);
```

Comment:
The arguments for these functions need to be updated.

Also I think `functions.h` could use a formatter run after these changes